Automattic/sensei
# Update e2e tests with the new course outline selection

Resolves #7012 

## Proposed Changes
* Update e2e tests to support the new course outline block selection screen
* Disable all other available plugins before running the e2e tests
* Set the WordPress to "development" to enable all feature flags.
* Use [random values to generate courses/categories](https://github.com/Automattic/sensei/pull/7011/files#diff-6f8a2d1cd1152d1fec5ef0ce267159d2bbff94f10aab5f3ec3d13b3159a3518aR21-R33), avoiding errors when the playwright tries to rerun a e2e test, but the course/category already exists on the database.
* Update the `npm run test:e2e` to ignore the tests with @setup tag.
* Add a new `npm run test:e2e:setup-only` to only run the tests with the @setup tag. 

## Testing Instructions
1. Check if the e2e tests are green
2. Run `npm run test:e2e`
3. Run `npm run test:e2e:setup-only`


## Pre-Merge Checklist
